Index.html

```diff
@@ -1,269 +1,176 @@
-                    <i class="fa fa-solid fa-arrow-left"></i>
-                    previous project
-                </button>
-                <button class="buttons__inverse nav__pop__next nav__pop__down">
-                    next project
-                    <i class="fa fa-light fa-arrow-right"></i>
-                </button>
-            </div>
-        </section>
-    </main>
-
-    <footer class="footer">
-        <button class="buttons footer__button">
-            <h4 class="footer__button__text">Get my resume</h4>
-            <img class="footer__button__image" src="./Contents/Resources/Icons/download_Resume.svg" alt="download my resume">
-        </button>
-
-        <ul class="socials">
-            <li href=""><img src="./Contents/Resources/Icons/ic_stackoverflow.svg" alt="go to my stackoverflow" class="social"></li>
-            <li href=""><img src="./Contents/Resources/Icons/ic_github.svg" alt="go to my github" class="social"></li>
-            <li href=""><img src="./Contents/Resources/Icons/ic_twitter.svg" alt="go to my twitter" class="social"></li>
-            <li href=""><img src="./Contents/Resources/Icons/ic_linkedin.svg" alt="go to my linkendin" class="social"></li>
-            <li href=""><img src="./Contents/Resources/Icons/ic_medium.svg" alt="go to my M" class="social"></li>
-            <li href=""><img src="./Contents/Resources/Icons/fingers.svg" alt="go to my M" class="social"></li>
+      </li>
+      <li>Frameworks
+        <ul id="frameworks">
+          <li>Bootstrap</li>
+          <li>Django</li>
+          <li>Rest</li>
+          <li>Celery</li>
+          <li>Postgres</li>
+        </ul>
+      </li>
+      <li>Skills
+        <ul id="skills">
+          <li>Problem solving</li>
+          <li>Agile</li>
+          <li>GitHub</li>
+          <li>GitLab</li>
+          <li>Terminal</li>
         </ul>
-    </footer>
+      </li>
+    </ul>
+  </section>
+  <form action="https://formspree.io/f/xeqwnkjg" method="POST" id="contact-form">
+    <h1>Get Started</h1>
+    <p>I prefer to speak by my gestgures.
+    </p>
+    <fieldset>
+      <label for="customer-name">
+        Your name <span class="errorText"></span>
+        <br />
+        <input id="customer-name" name="customer-name" type="text" placeholder="Full name" maxlength="30" required />
+      </label>
+    </fieldset>
+    <fieldset>
+      <label for="customer-email">
+        Your email <span class="errorText"></span>
+        <br />
+        <input id="customer-email" name="customer-email" type="email" placeholder="something@domain.com" required />
+      </label>
+    </fieldset>
+    <span class="errorText"></span>
+    <textarea id="customer-message" name="customer-message" maxlength="500" placeholder="Enter your message here..."
+      required></textarea>
+    <span class="errorText" id="submitError"></span>
+    <button id="btn-submit" class="dark small" type="button" name="submit1">Start collaboration</button>
+  </form>
```

Comment:
- [ ]  Kindly as requested by the previous reviewer consider making sure that you stick to the Figma design as much as possible mainly here am talking about your mobile version consider making sure that as you develop the desktop version make sure that the mobile version is not messed up that will make your work more professional
check on this part
your design 👇
<img width="321" alt="Screen Shot 2023-05-04 at 21 39 18" src="https://user-images.githubusercontent.com/60146030/236312532-c61fde1b-b351-4e4d-a1e8-df9b14060e9d.png">

figma design 👇 
<img width="291" alt="Screen Shot 2023-05-04 at 21 42 03" src="https://user-images.githubusercontent.com/60146030/236312734-42077980-cddd-45dd-bf21-a6d168705665.png">